app/core/base.js

```diff
@@ -157,6 +157,10 @@ FauxtonAPI.setSession = function (newSession) {
   return FauxtonAPI.session.fetchUser();
 };
 
-FauxtonAPI.reducers = [];
+FauxtonAPI.reducers = {};
+
+FauxtonAPI.addReducers = (reducers) => {
+  FauxtonAPI.reducers = Object.assign({}, FauxtonAPI.reducers, reducers);
+};
 
```

Comment:
This is really cool. What about if you used the [spread operator](https://babeljs.io/docs/plugins/transform-object-rest-spread/) here. Then this should be:

```
FauxtonAPI.addReducers = (reducers) => {
  FauxtonAPI.reducers = {
   ...FauxtonAPI.reducers,
   reducers
 };
```